runtime/onert/api/nnfw/src/nnfw_session.cc

```diff
@@ -986,7 +1021,18 @@ NNFW_STATUS nnfw_session::set_signature_for_tensorinfo(const char *signature)
     return NNFW_STATUS_INVALID_STATE;
   }
 
-  std::cerr << "Error during nnfw_session::set_signature_for_tensorinfo : NYI" << std::endl;
+  for (auto &sig : _signature_map)
```

Comment:
(Optional)
It would be nice to use structure bindings and `const`.

Reply:
I've updated